Source/Details/ASThread.h

```diff
@@ -9,11 +9,8 @@
 
 #import <Foundation/Foundation.h>
 
-#import <assert.h>
 #import <os/lock.h>
 #import <pthread.h>
```

Comment:
Note: pthread is still imported for ASDisplayNodeThreadIsMain function (no "main thread" in libcxx).